kuka_kr5_support/CMakeLists.txt

```diff
@@ -1,15 +1,17 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.16)
 
-project(kuka_kr5_support)
+project(kuka_kr5_support LANGUAGES CXX)
 
-find_package(catkin REQUIRED)
+if(CMAKE_CXX_COMPILER_ID MATCHES "(GNU|Clang)")
+  add_compile_options(-Wall -Wextra)
+endif()
 
-catkin_package()
+find_package(ament_cmake REQUIRED)
 
-if (CATKIN_ENABLE_TESTING)
-  find_package(roslaunch REQUIRED)
-  roslaunch_add_file_check(test/roslaunch_test.xml)
-endif()
```

Comment:
We should keep this test, but port it to ROS 2. I don't have a good example how to test launch files in ROS 2. Try to find an approach.

Reply:
We will do this in the follow-up PR